src/tests/venues/[id].test.jsx

```diff
@@ -20,52 +21,47 @@ const getMocks = () => [
     },
     result: {
       data: {
-        venue: {
-          addressLocality: "Helsinki",
-          dataSource: "",
-          description: "",
-          email: "",
-          id,
-          image: "",
-          infoUrl: "https://hel.fi",
-          name: "Eiran uimaranta",
-          position: {
-            type: "Point",
-            coordinates: [1, 2],
-          },
-          postalCode: "00001",
-          streetAddress: "Eirantie 3",
-          telephone: "+35812345678",
-          openingHours: getOpeningHoursForWeek(),
-          isOpen: false,
-          ontologyTree: [],
-          ontologyWords: [],
-          accessibilitySentences: [
-            {
-              groupName: "Pääsisäänkäynti",
-              sentences: ["Erottuu selkeästi", "Ahdas tuulikaappi"],
-            },
-          ],
-        },
+        venue: getVenue({ id }),
       },
     },
   },
 ];
 
-describe("venues/[id]", () => {
-  it("renders without crashing", async () => {
-    await act(async () => {
-      render(<VenuePageContent />, getMocks(), {
-        query: {
-          id,
-        },
-      });
+test("venues/[id] renders correctly", async () => {
+  await act(async () => {
+    render(<VenuePageContent />, getMocks(), {
+      query: {
+        id,
+      },
     });
-
-    await waitFor(() =>
-      expect(
-        screen.getByText("Eiran uimaranta", { selector: "h2" })
-      ).toBeInTheDocument()
-    );
   });
+
+  await waitFor(() =>
+    expect(
+      screen.getByText("Eiran uimaranta", { selector: "h2" })
+    ).toBeInTheDocument()
+  );
```

Comment:
I would prefer
`await screen.findByText("Eiran uimaranta", { selector: "h2" })`
here to keep it little bit more readable